cextern/expat/configure

```diff
@@ -1,13 +1,11 @@
 #! /bin/sh
 # Guess values for system-dependent variables and create Makefiles.
-# Generated by GNU Autoconf 2.68 for expat 2.1.0.
```

Comment:
Do we actually need to include this file? Because it's also an auto-generated file, I wonder if we might be able to do without if we don't actually need it?